code/modules/food/beverages/bottler.dm

```diff
@@ -0,0 +1,413 @@
+
+//adjust these to change the maximum capacity of the bottler for each container type
+#define MAX_GLASS 10
+#define MAX_PLAST 20
+#define MAX_METAL 25
+
+//adjust these to change the number of containers the bottler will make per sheet
+#define RATIO_GLASS 1
+#define RATIO_PLAST 2
+#define RATIO_METAL 5
+
+/obj/machinery/bottler
+	name = "bottler unit"
+	desc = "A machine that combines ingredients and bottles the resulting beverages."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "bottler_off"
+	density = 1
+	anchored = 1
+	var/list/slots[3]
+	var/list/datum/bottler_recipe/available_recipes
+	var/list/acceptable_items
+	var/list/containers = list("glass bottle" = 10, "plastic bottle" = 20, "metal can" = 25)
+	var/bottling = 0
+
+/obj/machinery/bottler/New()
+	if(!available_recipes)
+		available_recipes = new
+		acceptable_items = new
```

Comment:
Lists like these should be initialized as `= list()`, so it's obvious they're lists and not some other type that actually requires `new`ing. Also there might be some weird performance thing related to using `list()`? I dunno.